### ISSUE TITLE
ESLint double dependencies

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/ethereumjs-config#42]
Currently `eslint` as well as other plugins like `typescript-eslint/eslint-plugin` are listed both in the consuming library [package.json](https://github.com/ethereumjs/ethereumjs-vm/blob/master/packages/vm/package.json) files (as `devDependencies`) as well as the `lint` config package [package.json](https://github.com/ethereumjs/ethereumjs-config/blob/master/packages/lint/package.json) (as dependencies).

This is inconsistent and confusing (also on the fact which version is then actually used since version numbers significantly differ) and there should be some clear decision and subsequent description in the docs where these dependencies should be located.

**Comment by rumkin** [comment]
I've reviewed this repo `package.json` and found the eslint in dependencies. And now I understand what was confusing in my question on the call. I'm using PNPM and it doesn't install binaries from nested dependencies. This is why I've got missing eslint error with ethereumjs-config-lint command call. As a solution I can install eslint globally or into some of parent directories. Don't know if it should be added as an independent issue or just be noted here. In the future it could be a reason of new issues on this.
